Schutzfile

```diff
@@ -3,7 +3,7 @@
   "fedora-39": {
     "dependencies": {
       "osbuild": {
-        "commit": "dfd59a38d9ec7458944f9a901a6aa6e1202a7efb"
+        "commit": "25df4d76a6da3b75fdb6f48ae48e27f31a15a38b"
```

Comment:
Isn't this needed for all distros on which we test OVA? :thinking:

Reply:
We only have one runner defined in the Schutzfile here.  All tests run on Fedora 39.

Reply:
Thanks. I should have checked that myself. :roll_eyes: Anyway, we should be probably testing also on RHEL.